worker/on-premise/docker-registry.md

```diff
@@ -0,0 +1,48 @@
+---
+title: IronWorker On-Premise Docker Registry
+layout: default
+section: worker
+breadcrumbs:
+  - ['On-Premise', '/on-premise']
+---
+
+This will allow you to use a custom docker registry with an Iron On-Premise installation.
+
+There are two types of docker registries, insecure and secure. The one which you configure depends on your situation. However, due to limitations on controlling domains and certificates in an On Premise environment, we are only able to create an insecure registry.
+
+# Installation
+
+The only difference in workflow between an insecure registry and a secure registry is that you'll need to whitelist the registry from your local docker daemon.
+
+To set up a registry with the On Premise Installer, simply include a `[docker-registry]` host in your `./config/inventory.ini` with a proper server IP and the docker registry will be installed on that server.
+
+```
+[docker-registry]
+192.168.x.x
+```
+
+This registry will serve on port `6000`.
+
+# Usage
+
+You will need to alter your local docker before being able to push images to this registry. This is a little different depending on your OS -- please see [here](https://docs.docker.com/registry/insecure/) for how to set it up.
```

Comment:
Nah, our installer doesn't take care of this. In order to push/pull to an insecure registry one has to whitelist the Docker registry from their local Docker daemon on each machine they plan to push/pull with. It's not something that we can add into the installer.